src/schema/index.js

```diff
@@ -0,0 +1,276 @@
+  'restrictions on ballistic missiles',
+  'embargo on natural resources',
+  'non-proliferation measures',
+  'oil/petroleum embargo',
+];
+
+const measureTypes = [
+  'establish',
+  'modify',
+  'extend',
+  'terminate',
+  'limited extend',
+];
+
+const measureSchema = new Schema({
+  measureCategory: requiredEnum(measureCategories),
+  measureType: requiredEnum(measureTypes),
+});
+
+const decisionTypes = [
+  'extend',
+  'implementation',
+  'establish',
+  'exemption',
+  'intention',
+  'terminate',
+];
+
+const decisionSchema = new Schema({
+  decision: required(String),
+  regime: required(String),
+  year: required(Number),
+  date: required(Date),
+  numParagraphs: required(Number),
+  decisionType: requiredEnum(decisionTypes),
+  measures: [measureSchema],
+});
+
+const ccrTableNames = ['wps', 'caac', 'poc'];
+const ccrCategories = ['thematic', 'country/region'];
+const ccrStatementTypes = ['pp', 'op', 'prst'];
+const ccrSchema = new Schema({
+  table: requiredEnum(ccrTableNames),
+  symbol: required(String),
+  category: requiredEnum(ccrCategories),
+  agendaItem: required(String),
+  statementType: requiredEnum(ccrStatementTypes),
+  paragraphId: required(String),
+  provision: required(String),
+  keywords: [String],
+});
+
+const mandateComponents = [
+  'Rule of law (ROL)',
+  'Humanitarian support',
+  'Public information',
+  'Political process',
+  'Electoral assistance',
+  'Support to sanctions regimes',
+  'Military and police',
+  'Support to State institutions',
+  'Security sector reform (SSR)',
+  'Human rights, women and peace and security, and children and armed conflict',
+  'International cooperation and coordination',
+  'Civilian-military coordination',
+  'Demilitarization and arms management',
+  'Authorization of the use of force',
+];
+const mandateSubcomponents = [
+  'Judicial matters',
+  'Protection of civilians, including refugees and IDPs',
+  ':DPA-led',
+  'Support to military',
+  'Ceasefire monitoring',
+  ':DPKO-led',
+  'Security monitoring - patrolling - deterrence',
+  'Protection of humanitarian/UN personnel and facilities / free movement of personnel and equipment', // eslint-disable-line max-len
+  'Support to police',
+  'Maritime security',
+];
+const mandateComponentSchema = new Schema({
+  component: requiredEnum(mandateComponents),
+  subcomponent: optionalEnum(mandateSubcomponents),
+  resolutions: String,
+  excerpt: String,
+});
+
+const leadEntities = ['DPKO', 'DPA', 'DPKO/AU'];
+const mandateSchema = new Schema({
+  name: required(String),
+  location: String,
+  originalDecision: required(String),
+  subsequentDecisions: [String],
+  latestDecision: String,
+  expiration: Date,
+  currentLength: required(String),
+  leadEntity: requiredEnum(leadEntities),
+  chapterVII: required(Boolean),
+  authorizationOfUseOfForce: mandateComponentSchema,
+  mandateComponents: [mandateComponentSchema],
+});
+
+const graphqlSchema = makeExecutableSchema({
+  typeDefs: `
+    type Mutation {
```

Comment:
I have been using a babel plugin called [inline-import](https://www.npmjs.com/package/babel-plugin-inline-import) over on another gql project that is pretty nice, might be something to add going forward but not a merge blocker.